tests/src/Commands/FirstCustomCommands.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Robo\Commands;
```

Comment:
This should probably be in some namespace other than the default pattern, so that tests must explicitly set the discovery namespace in order for these to be discovered. We don't want to pollute all of the functional tests with the plugin fixtures.